packages/react-composites/src/composites/CallComposite/adapter/CallAdapter.ts

```diff
@@ -247,6 +252,64 @@ export interface VideoBackgroundImage {
   tooltipText?: string;
 }
 
+/* @conditional-compile-remove(video-background-effects) */
+/**
+ * Contains the attibutes of a selected video background effect
+ *
+ * @beta
+ */
+export type SelectedVideoBackgroundEffect =
+  | VideoBackgroundNoneEffect
+  | VideoBackgroundBlurEffect
+  | VideoBackgroundReplacementEffect;
+
+/* @conditional-compile-remove(video-background-effects) */
+/**
+ * Contains the attibutes to remove video background effect
+ *
+ * @beta
+ */
+export interface VideoBackgroundNoneEffect {
```

Comment:
Do we need two types here? could we just  have `VideoBackgroundBlurEffect` where `effectName: 'blur' | 'none'` is the property?

Reply:
Yeah, so none effect is basically to remove any video effects that includes both `blur` and `replacement` effects. If we add `none` to `VideoBackgroundBlurEffect ` that is kind of misleading that blur that selecting none would remove blur effect, whereas we want to keep it as separate type to remove either of two effects.